bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/thing/ozw672.xml

```diff
@@ -0,0 +1,111 @@
+				<context>network-address</context>
+			</parameter>
+			<parameter name="userName" type="text">
+				<description>User name of the Siemens Hvac gateway</description>
+				<required>false</required>
+				<label>userName</label>
+				<default>Administrator</default>
+			</parameter>
+			<parameter name="userPassword" type="text">
+				<description>User password of the Siemens Hvac gateway</description>
+				<required>false</required>
+				<label>userPassword</label>
+				<default>password</default>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+
+
+
+
+	<channel-type id="outside-temp">
+		<item-type>Number:Temperature</item-type>
+		<label>Outside temp</label>
+		<description>Outside temp</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.1">
+		</state>
+	</channel-type>
+
+
+	<channel-type id="setpoint-heat-pump">
+		<item-type>Number:Temperature</item-type>
+		<label>Setpoint heat pump</label>
+		<description>Setpoint heat pump</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.1">
+		</state>
+	</channel-type>
+
+
+	<channel-type id="room-temp-comfort-setpoint-hc">
+		<item-type>Number:Temperature</item-type>
+		<label>RoomTemp ComfSetp HC</label>
+		<description>Room temperature Comfort setpoint HC</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.5">
+		</state>
+	</channel-type>
+
+	<channel-type id="room-temp-reduce-setpoint-hc">
+		<item-type>Number:Temperature</item-type>
+		<label>RmTmp RedSetp HC</label>
+		<description>Room temp reduced setpoint heat circuit</description>
+		<category>Temperature</category>
+		<state readOnly="true" pattern="%.1f %unit%" step="0.5">
+		</state>
+	</channel-type>
+
+	<channel-type id="operating-mode-hc">
+		<item-type>Number</item-type>
+		<label>OptgMode HC</label>
+		<description>Operating mode heat circuit</description>
+		<state>
+			<options>
+				<option value="0">Protection mode</option>
+				<option value="1">Automatic mode</option>
+				<option value="2">Reduced mode</option>
+				<option value="3">Comfort mode</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="status-hc2">
+		<item-type>Number</item-type>
+		<label>Status</label>
+		<description>Status heating circuit</description>
+		<category>Temperature</category>
+		<state>
+			<options>
+				<option value="0">---</option>
+				<option value="1">STB interrogé</option>
+				<option value="2">Défaut</option>
+				<option value="3">Contrôleur interrogé</option>
+				<option value="4">Régime manuel actif</option>
+				<option value="5">Fonction ramonage, pleine charge</option>
+				<option value="6">Fonction ramonage, charge partielle</option>
+				<option value="7">Fonction ramonage active</option>
+				<option value="8">Bloqué, manuel</option>
+				<option value="9">Bloqué, automatique</option>
```

Comment:
Can you provide this in English?

After that you can generate an I18N properties file - see:
https://www.openhab.org/docs/developer/utils/i18n.html#generating-i18n-properties-file

When the PR is merged, you can then translate into French in Crowdin.